src/main/scala/com/databricks/spark/sql/perf/tpcds/TPCDS_Standalone.scala

```diff
@@ -0,0 +1,276 @@
+package com.databricks.spark.sql.perf.tpcds
+
+import com.typesafe.scalalogging.slf4j.{LazyLogging => Logging}
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, LocalFileSystem, Path}
+import org.apache.spark.deploy.SparkHadoopUtil
+import org.apache.spark.sql.SparkSession
+
+import com.databricks.spark.sql.perf.RunBenchmark
+
+case class TpcdsStandaloneConfig(
+  sparkMaster: String = "local[*]",
+  datasetLocation: String = null,
+  outputLocation: String = null,
+  scaleFactor: Int = 10,
+  iterations: Int = 1,
+  regenerateDataset: Boolean = false,
+  useDecimal: Boolean = true,
+  useDate: Boolean = true,
+  filterNull: Boolean = false,
+  shuffle: Boolean = true,
+  timeoutHours: Int = 60,
+  format: String = "parquet",
+  shufflePartitions: Int = 12,
```

Comment:
Why 12?
Seems like a bad idea to set it that low.
Keeping the Spark default of 200 seems ok to me.